app.js

```diff
@@ -108,17 +109,30 @@ function countClicks(event) {
   for(var i = 0; i < pictureArray.length; i++){
     if(event.target.name === pictureArray[i].name)
       pictureArray[i].numberOfTimesClicked++;
+    // picturesStorageData();
       // console.log(pictureArray[i].numberOfTimesClicked);
   }
   if (totalNumberOfClicks < 25){
     generateNewImages();
+    picturesStorageData();
   }else{
     pictureBox.textContent = '';
     createChart();
-
   }
 }
 
+//function to save the stage of the puctures shows and Clicks
+function picturesStorageData() {
+
+  if( JSON.parse(localStorage.getItem('pictureStoredData')) == null) {
```

Comment:
This can be simplified as folllows:

```JavaScript
if (JSON.parse(localStorage['pictureStoredData'])) {
    // do stuff
}
```
or
```JavaScript
if (JSON.parse(localStorage.getItem('pictureStoredData'))) {
    // do stuff
}
```